src/server/status/states.js

```diff
@@ -18,11 +18,14 @@
  */
 
 import _ from 'lodash';
+import { i18n } from '@kbn/i18n';
 
 export const all = [
   {
     id: 'red',
-    title: 'Red',
+    title: i18n.translate('server.status.redTitle', {
```

Comment:
These labels are initialized statically at import time, so they will always be set with default locale. Please wrap `all` variable into function.